buildsystem/dependencies.gradle

```diff
@@ -28,7 +28,7 @@ ext {
     generalDependencies = [
             rxAndroid : "io.reactivex.rxjava2:rxandroid:${rxAndroidVersion}",
             rxJava    : "io.reactivex.rxjava2:rxjava:${rxJavaVersion}",
-            appSupport: "com.android.support:appcompat-v7:${appSupportVersion}",
+            appSupport: 'androidx.appcompat:appcompat:1.0.0',
```

Comment:
if you will use androidx please change the name "appSupport" it is not any more android.support

Reply:
done